src/tests/WalletPaymentsContainer.spec.tsx

```diff
@@ -69,7 +69,7 @@ describe('WalletPaymentsContainer', () => {
           await waitFor(async () => {
             wrapper = await renderWithProvider(<InitTestComponent />, {
               liveMode: false,
-              styleProps: { googlePayButton: { buttonBorderRadius: 8 } },
+              styleProps: { showSupportedCards: false, googlePayButton: { buttonBorderRadius: 8 } },
```

Comment:
prevent img preview from rendering for other tests, because fake timer is required for Animation tests